doc/cookbook/source/index.rst

```diff
@@ -105,3 +105,13 @@ Neural Networks
    :caption: Neural Netwroks
 
    examples/neural_nets/**
+
+Feature Transformations
```

Comment:
I guess we call these things "converter" internally.
Maybe just leave it as is now, easy to change later